pkg/flowaggregator/exporter/ipfix.go

```diff
@@ -169,6 +169,19 @@ func (e *IPFIXExporter) sendRecord(record ipfixentities.Record, isRecordIPv6 boo
 			return fmt.Errorf("error when initializing IPFIX exporting process: %v", err)
 		}
 	}
+	templateID := e.templateIDv4
+	if isRecordIPv6 {
+		templateID = e.templateIDv6
+	}
+	// This step is necessary because the templateID used by this exporter may not match the one
+	// from the record that we received.
```

Comment:
Does it mean the exporter in FA always override the templateID from flow exporter in agent?